src/constants/index.ts

```diff
@@ -0,0 +1,9 @@
+export const DEFAULT_DOMAIN = 'meet.jit.si';
+
+export const DEFAULT_OPTIONS = {
+    roomName: 'DefaultRoomNameExample',
+    width: '100%',
```

Comment:
Let's drop width and height, the iframe should cover 100% of the container we create and then the embedder can customize the container with the ref.